packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -883,10 +883,15 @@ export class AllocationReceiptCollector implements ReceiptCollector {
 
         try {
           const addressWithoutPrefix = rav.allocationId.toLowerCase().replace('0x', '')
-          await this.models.receiptAggregateVouchers.update(
-            { redeemedAt: new Date() },
-            { where: { allocationId: addressWithoutPrefix } },
-          )
+          // WE use sql directly due to a bug in sequelize update:
```

Comment:
Did this change since my last review?

Reply:
Yes this was also causing the same bug as before, I just missed this section too until I hit the error in here too